document/chapters/04-simd_bvh.tex

```diff
@@ -635,32 +635,37 @@ \subsubsection{Reduced-Precision Ray Tracing}\label{subsubsec:reduced_precision}
 can be described as
 
 \begin{equation}
-  \Delta = \sqrt{ (R_{u} - r_{u})^{2} + (R_{v} - r_{v})^{2} + (R_{w} -
-    r_{w})^{2} }
+  \begin{split}
+    \Delta = \sqrt{(R_{x} - r_{x})^{2} + (R_{y} - r_{y})^{2} + (R_{z} - r_{z})^{2})} + \\ \sqrt{ (R_{u} - r_{u})^{2} + (R_{v} - r_{v})^{2} + (R_{w} -
```

Comment:
The R(x,y,z)/r(x,y,z) delta can be significant for values far from the origin, but (for this description) the l_max should capture that. There is also a small-model assumption recognized here.

For the case of large models, the limitation for this method is still the maximum distance the ray can travel for a given query I think. The large deltas from R(x,y,z)/r(x,y,z) can be mitigated by using local origins to limit the magnitude of these values.

Reply:
l_max could be small for a small volume that is centered far from the origin.

Reply:
In this form, l_max is based on the total model size so hopefully other volumes in the model would increase l_max to an appropriate size... which is kind of a moot point because performance would certainly start to suffer.

However, it's possible that a model may be made entirely of small volumes far (extremely far) from the origin, keeping the value of l_max small. Maybe that's what you were driving at? In which case I would recommend moving the entire model closer to the problem origin.